docs/surveys/2020-jupyterlab-survey.md

```diff
@@ -0,0 +1,374 @@
+# JupyterLab
+
+ - Even if you don't use Jupyter, you can still take this survey. Just indicate this fact in the first question 
+   and carry on as best as you can.
+ - Thank you. Your participation guides Jupyter's roadmap toward your real-life use cases by quantifiably 
+   helping us prioritize the functionality that is important to our userbase. 
+ - So that you know what to expect, it's comprised of 20 questions spread across the sections below. 
+   As a fair heads up, Question 7  is the biggest one, but it provides critical information.
+    - Usage patterns.
+    - Data
+    - Visualization.
+    - Scale.
+    - Collaboration.
+ - The aggregate survey data itself will be openly shared with the Jupyter community when polling closes in mid-December.
+   If you opt to provide your email address for a user interview, it not be used for Jupyter's promotional purposes and 
+   it will not be shared with a 3rd party.
+
+
+### Usage Patterns
+
+1. How frequently do you use Jupyter?
+    - Daily - heavy usage; 3+ hours per day.
+    - Daily - moderate usage; less than 3 hours per day.
+    - Weekly.
+    - Monthly.
+    - I no longer use Jupyter.
+    - I have never used Jupyter.
+
+2. How long have you been using Jupyter?
+    - 2+ years.
+    - 1-2 years.
+    - 6-12 months.
+    - Less than 6 months (welcome =]).
+    - I don't use Jupyter.
+
+3. What languages do you use in Jupyter? (pick up to 4)
+    - C (and derivatives)
+    - Go
+    - Groovy
+    - Java
+    - JavaScript
+    - Julia
+    - NodeJS
+    - Perl
+    - PHP
+    - Python
+    - R
+    - Ruby
+    - Rust
+    - Scala
+    - Spark SQL
+    - SQL
+    - TypeScript
+    - ❗I wrap/ use bindings for other languages.
+    - ❗My preferred language is not supported in Jupyter.
+    - Other (please specify)
+
+4. What are your primary job roles when you are using Jupyter? (pick up to 2)
+    - Backend engineer.
+    - Business analyst.
+    - Data engineer.
+    - Data scientist.
+    - Database Admin (DBA).
+    - DevOps.
+    - Financial modeler/ analyst.
+    - Front end/ web development.
+    - Infrastructure engineer/ cloud architect.
+    - Scientist/ researcher.
+    - Student.
+    - Sysadmin.
+    - Teacher/ lecturer.
+    - Tutor/ teaching assistant.
+    - Other (please specify)
+
+5. What are your go-to tools for performing data science, scientific computing, 
+and machine learning on your laptop/ desktop (non-cloud) for data science? (pick up to 3)
+    - Atom.
+    - Emacs.
+    - iPython.
+    - Jupyter Notebook - Classic.
+    - JupyterLab.
+    - nteract.
+    - PyCharm.
+    - RStudio.
+    - Spyder.
+    - Sublime Text.
+    - Vim.
+    - VS Code.
+    - Zeppelin.
+    - Other (please specify).
+
+6. How do you run and/ or access Jupyter? (pick up to 4)
```

Comment:
I believe this is one of those questions that the alphabetical sort order might not be best, as a great percentage of users might be using Jupyter in their local env of some sort which would be the last answer or towards the end? Having said that, I don't have strong feelings one way or another...